mzLib/Transcriptomics/RNA.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Chemistry;
+using Omics.Modifications;
+
+namespace Transcriptomics
+{
+    public class RNA : NucleicAcid
+    {
+        /// <summary>
+        /// For constructing RNA from a string
+        /// </summary>
+        /// <param name="sequence"></param>
+        /// <param name="fivePrimeTerm"></param>
+        /// <param name="threePrimeTerm"></param>
+        /// <param name="oneBasedPossibleLocalizedModifications"></param>
+        public RNA(string sequence, IHasChemicalFormula? fivePrimeTerm = null, IHasChemicalFormula? threePrimeTerm = null,
+            IDictionary<int, List<Modification>>? oneBasedPossibleLocalizedModifications = null)
+            : base(sequence, fivePrimeTerm, threePrimeTerm, oneBasedPossibleLocalizedModifications)
+        {
+        }
+
+        /// <summary>
+        /// For use with RNA loaded from a database
+        /// </summary>
+        /// <param name="sequence"></param>
```

Comment:
gene?

Reply:
good catch. Added gene name to RNA and Nucleic acid constructor as well as RNA database parsing